sqlmesh/integrations/dlt.py

```diff
@@ -130,7 +162,7 @@ def generate_incremental_model(
 {select_columns},
   {time_column} as _dlt_load_time
 FROM
-  {from_table}
+  {from_table} as c{join}
```

Comment:
[Nit] should we remove the leading `\n` from L146 and just put a newline here instead?